### PR TITLE
Add some words to glossary

### PR DESCRIPTION
**Pull request by Blind4Basics**
Starting point, to see how it behaves.

I didn't update the stump that was already there (dan/kyus/...)

note @kazk : what do you expect to write about TDD considering the `TODO` note that is there?

### CODE REVIEW

**Comment by Blind4Basics** [comment]
#23

**Comment by Blind4Basics** [comment]
...okay... I totally failed the formatting... x)
Surely the multiline description that are wrong. I'll see about that tomorrow (if anyone has the answer on hand, do not hesitate to tell me... x) )

**Comment by kazk** [comment]
> note @kazk : what do you expect to write about TDD considering the TODO note that is there?

Nothing. That was just an example to show you guys that Markdown can be used :)

File: data/glossary.yml [hunk 1]
```diff
@@ -31,12 +31,86 @@
...
+  description: User(s) hwo have either authored, approved, translate or modify a kata in any way (a list of them can be seen in the "Details" page of any kata.
+
 - term: Dan
   description: 段
   links:
     - title: Dan (rank) - Wikipedia
       url: https://en.wikipedia.org/wiki/Dan_(rank)
 
+- term: [Dashboard](https://www.codewars.com/dashboard)
+  description: This is the main page for a code warrior. You land there by clicking on the picture in the top left vertical bar (CodeWars' logo). It contains kata suggestions, a leaderboard of you and your allies and a flow of the last comments posted anywhere on CodeWars.
```
Comment (kazk) [inline data/glossary.yml:67]:
```suggestion
- term: Dashboard
  description: This is the main page for a code warrior. You land there by clicking on the picture in the top left vertical bar (CodeWars' logo). It contains kata suggestions, a leaderboard of you and your allies and a flow of the last comments posted anywhere on CodeWars.
  links:
    - title: Codewars Dashboard
      url: https://www.codewars.com/dashboard
```

File: data/glossary.yml [hunk 2]
```diff
@@ -31,12 +31,86 @@
...
+- term: Example tests
+  description: |
+    Tests that are visible to the user in the trainer. Most of the time, they are only a tiny part of the full tests suite. Hence, passing all the tests of the example tests doesn't make you pass the kata. You have to use `ATTEMPT`.
+    You run the example tests using the `TEST` button in the trainer.
+
+- term: Fixed tests
+  description: |
+    Tests whose input and output are always the same, each time your run the tests. Generally, the example tests are made of those, and the test cases contains both fixed and random tests.
+
+- term: Followers.
```
Comment (kazk) [inline data/glossary.yml:88]:
```suggestion
- term: Followers
```

File: data/glossary.yml [hunk 3]
```diff
@@ -31,12 +31,89 @@
...
       url: https://en.wikipedia.org/wiki/Dan_(rank)
 
+- term: Dashboard
+  description: This is the main page for a code warrior. You land there by clicking on the picture in the top left vertical bar (CodeWars' logo). It contains kata suggestions, a leaderboard of you and your allies and a flow of the last comments posted anywhere on CodeWars.
+  links:
+    - title: Codewars Dashboard
+      url: https://www.codewars.com/dashboard
+
+- term: Draft (kata)
+  description: Initial state of a new born kata: when you create a new kata, it is in Draft mode as long as it is not "published". Katas in draft state are not visible through the [katas search page](https://www.codewars.com/kata/my-languages), but can be accessed and tried by other users if the author gives them the direct link to the trainer. Note that a kata in draft state cannot be completed (eg. "submit final" will not show up if you pass all the tests of a draft).
```
Comment (kazk) [inline data/glossary.yml:73]:
```suggestion
  description: |
    Initial state of a new born kata: when you create a new kata, it is in Draft mode as long as it is not "published". Katas in draft state are not visible through the [katas search page](https://www.codewars.com/kata/my-languages), but can be accessed and tried by other users if the author gives them the direct link to the trainer. Note that a kata in draft state cannot be completed (eg. "submit final" will not show up if you pass all the tests of a draft).
```

**Comment by kazk** [comment]
It was failing because you had `:` in unquoted string which is a reserved character.
Quoting a string is optional in YAML, but you can run into issues like this.

File: data/glossary.yml [hunk 4]
```diff
@@ -31,12 +31,89 @@
...
+    In the trainer, when you click on the `ATTEMPT` button, you run your code against the full test suite (see "test cases"). Validating all the tests makes you pass/validate the kata, and grants you honor points and progress.
+
+- term: Author
+  description: User who created a kata (either an approved one or one still in beta).
+
+- term: Beta (kata/process)
+  description: The beta process is the phase where a kata is visible to any other user in the database/[katas search page](https://www.codewars.com/kata/my-languages) but does not have a rank yet. It is to be tested by other users, so that it can be enhanced, modified, ranked and finally approved... or retired if not qualitative enough.
+
+- term: Contributor(s)
+  description: User(s) hwo have either authored, approved, translate or modify a kata in any way (a list of them can be seen in the "Details" page of any kata.
```
Comment (kazk) [inline data/glossary.yml:57]:
```suggestion
  description: User(s) who have either authored, approved, translated or modified a kata in any way (a list of them can be seen in the "Details" page of any kata.
```

**Comment by Blind4Basics** [comment]
x/

Damn it... The linter swallows it, but the other verifications are failing and I don't get why...

* no colons
* I tried removing the `/` in the `term` section of `Approve (kata/translation)`
* I just corrected a typo and exchanged 2 terms that weren't in alphabetic order

... => ?? :(

**Comment by kazk** [comment]
Other checks failed because of some cache issues on Netlify Build. Probably related packages on this branch not matching the one on master because I updated some few days ago.

I rebuilt after clearing the cache and it’s fine now.

Don’t merge this yet. I’ll squash merge later because commits are messy.

File: data/glossary.yml [hunk 5]
```diff
@@ -31,12 +31,91 @@
...
+  description: |
+    Author of a kata who hasn't been connected for more than 30 days. When an author is inactive, some actions he's normally the only person allowed to do are doable by other people (power users with the required privilegees).
+
+- term: Initial solution
+  description: |
+    This is the starting point of your code when you enter the trainer of a kata (the "Solution" box).
+
+- term: Issue (comment)
+  description: |
+    `Issue is the label to use when you write a comment in the discourses section of a kata to point out some problems in the kata itself (wrong tests, wrong reference solution, wrong description, missing tests, misleading description, ...).
```
Comment (kazk) [inline data/glossary.yml:116]:
```suggestion
    `Issue` is the label to use when you write a comment in the discourses section of a kata to point out some problems in the kata itself (wrong tests, wrong reference solution, wrong description, missing tests, misleading description, ...).
```

File: data/glossary.yml [hunk 6]
```diff
@@ -59,10 +138,27 @@
...
+
+- term: Question (comment)
+  description: |
+    `Question` is the label to use when you write a comment in the discourses section of a kata when you encounter some problems to solve it and you don't understand why your solution is failing some (or all) the tests. Or if you ___believe___ your solution is correct while the expected result isn't.
+    Most of the times, the user (you) is actually missing something, so use the Question label and not the Issue label, unless you can ___prove___ that the problem you're facing is actually an "issue".
+
 - term: Test Driven Development
   description: |
+    Katas are built following this principle: you are provided with specifications and a set of tests, and your goal is to pass those tests (by following the specifications or finding a way to at least pass the tests).
     > **TODO** Explain. Include why TDD is relevant to Codewars.
```
Comment (kazk) [inline data/glossary.yml:152]:
```suggestion
```

**Pull request merged**